curvefs/src/client/rpcclient/mds_client.cpp

```diff
@@ -550,7 +552,17 @@ MdsClientImpl::RefreshSession(const std::vector<PartitionTxId> &txIds,
             FsQuotaChecker::GetInstance().UpdateQuotaCache(
                 response.fscapacity(), response.fsusedbytes());
         }
-
+        if (response.has_mdsaddrsoverride()) {
+            std::vector<std::string> mdsAddrsOverride = {};
+            curve::common::SplitString(response.mdsaddrsoverride(), ",",
+                                       &mdsAddrsOverride);
+            if (!mdsAddrsOverride.empty()) {
+                mdsOpt_.rpcRetryOpt.addrs = mdsAddrsOverride;
```

Comment:
data race

Reply:
mdsOpt_.rpcRetryOpt in MdsClientImpl is only used by Init() seems, so no data race i think, will leave a comment to hint

Reply:
ok, will add safe get&set for rpcexcutor_'s retryOption